src/utils/management/CustomState.ts

```diff
@@ -1,34 +1,23 @@
-export class CustomState<T> {
-  private error: string;
-  private pending: boolean;
-  private data: T;
-  private readonly preparedData: T;
+export function State<T>(data: T): CustomState<T> {
```

Comment:
Replace old function syntax with new one - export const State<T> = (data: T): CustomState<T> => {} - we using in whole project arrow functions - so this breaks convention